Source/Tests/LinuxSyscalls/Syscalls.h

```diff
@@ -178,6 +188,8 @@ class SyscallHandler : public FEXCore::HLE::SyscallHandler {
   uint32_t HostKernelVersion{};
   uint32_t GuestKernelVersion{};
 
+  FEXCore::Context::Context *CTX;
```

Comment:
We don't repoint nor own this pointer, which would be more readily visible using a reference.

Reply:
Nowhere in the code we use references for context, and pretty much for any other object that is passed around, so I'd like to stick here with the existing convention there.

Tracking allocation/free responsibilities, as well as nullable pointers would be an interesting code quality improvement though, I'd be all for it as a separate PR.

Reply:
@neobrain created #1678 to continue this discussion there